src/app/containers/ChartbeatAnalytics/utils/index.js

```diff
@@ -146,7 +146,7 @@ export const getConfig = ({
     sections,
     uid: chartbeatUID,
     title,
-    virtualReferrer: decodeURIComponent(referrer),
+    virtualReferrer: referrer && decodeURIComponent(referrer),
```

Comment:
```suggestion
    virtualReferrer: referrer ? decodeURIComponent(referrer) : null,
```

Reply:
I impulsively committed this suggestion 👀 

but is this `virtualReferrer: referrer && decodeURIComponent(referrer)` not achieving the same. Since `referrer` is either a string or null, it would just return `null` if this condition `referrer && decodeURIComponent(referrer)` did not pass.

![image](https://user-images.githubusercontent.com/30599794/80406040-5d12da80-88bb-11ea-9b10-af1691ae8cc6.png)

Reply:
Ah! Yes, you are correct.  I assumed this would return `undefined` in the case where the expression evaluates as false. Sorry about that.